lib/ecto/migrator.ex

```diff
@@ -31,10 +31,10 @@ defmodule Ecto.Migrator do
   This function ensures the migration table exists
   if no table has been defined yet.
   """
-  @spec migrated_versions(Ecto.Repo.t) :: [integer]
-  def migrated_versions(repo) do
-    SchemaMigration.ensure_schema_migrations_table!(repo)
-    SchemaMigration.migrated_versions(repo)
+  @spec migrated_versions(Ecto.Repo.t, atom) :: [integer]
+  def migrated_versions(repo, prefix) do
```

Comment:
Let's pass the prefix here as part of options to be consistent with `up` and `down`. :)